src/main/java/seedu/allonus/contacts/ContactsManager.java

```diff
@@ -135,7 +145,7 @@ private static void deleteContact(String userInput) {
         isModified = true;
     }
 
-    private static void addContact(String userInput) {
+    private static void addContact(String userInput, boolean fromCommandLine) {
```

Comment:
Consider naming the boolean parameter such that it sounds more like a boolean, e.g. `isCommand`, `isFromCommandLine`, or something like `shouldNotPrint`